packages/selfcare-client-users-updater/package.json

```diff
@@ -6,7 +6,8 @@
   "main": "dist",
   "type": "module",
   "scripts": {
-    "test": "echo \"Old readmodel\" && vitest --config vitest.config.ts && echo \"SQL readmodel\" && vitest --config vitest.sql.config.ts",
+    "test": "echo \"SQL readmodel\" && vitest --config vitest.sql.config.ts",
+    "test:old": "echo \"Old readmodel\" && vitest --config vitest.config.ts ",
     "test:it": "vitest integration",
```

Comment:
I suggest to adopt a single approach to execute integration test task, currently we have `test:it` and `test:integration` task for the same purpose, but those tasks using a different criteria to find the file for execution.
I suggest to uniform all integration test task using configuration in file `vitest.integration.config.ts` as described [here](https://pagopaspa.slack.com/archives/C06D24MANNN/p1749634973717609).